theme/scripts/productDescription.ts

```diff
@@ -0,0 +1,83 @@
+export default(function() {
+	const ProductDescription = {
```

Comment:
Some strange formatting here, any chance we can format it a bit nicer? (doesn't look like we have prettier set up on this project, but if you are using VSCode or similar they have a local plugin which can handle it)

Reply:
I have Prettier setup but my settings must be different than what the repo uses (2 spaces vs 4 spaces, etc), I'll try to adjust this